swebench/harness/context_manager.py

```diff
@@ -341,7 +342,7 @@ def __enter__(self):
 
                     # Install dependencies
                     path_to_reqs = get_requirements(setup_ref_instance, self.testbed)
-                    cmd = f". {path_activate} {env_name} && echo 'activate successful' && pip install -r {path_to_reqs}"
+                    cmd = f"source {path_activate} {env_name} && echo 'activate successful' && {self.path_conda}/envs/{env_name}/bin/pip install -r {path_to_reqs}"
```

Comment:
was replacing `.` with `source` part of the fix? Because https://github.com/princeton-nlp/SWE-bench/pull/31/ states that `.` makes it more cross-platform (as `source` doesn't exist in some other shell variants but `.` does (and should be equivalent to `source` in bash))

Reply:
oh no that's just incidental.

Reply:
updated